layouts/shortcodes/docs/languages/libraries-intro.md

```diff
@@ -0,0 +1,48 @@
+<!-- prettier-ignore -->
+{{ $lang := .Get 0 -}}
+{{ $howMany := .Get 1 | default 10 -}}
+{{ $integrations := where (where $.Site.Data.ecosystem.integrations ".components" "intersect" (slice $lang)) ".native" "eq" true -}}
+
+When you develop an app, you make use of third-party libraries and frameworks to
+accelerate your work and to not reinvent the wheel. If you now instrument your
+app with OpenTelemetry, you don't want to spend additional time on manually
+adding traces, logs and metrics to those libraries and frameworks. Fortunately,
+you don't have to reinvent the wheel for those either: libraries might come with
+OpenTelemetry support natively or you can use an
+[Instrumentation Library](/docs/concepts/instrumentation/libraries/) in order to
+generate telemetry data for a library or framework.
+
+If you are instrumenting an app, you can learn on this page how to make use of
+natively instrumented libraries and Instrumentation Libraries for your
+dependencies.
```

Comment:
```suggestion
If you are instrumenting an application or service that use third-party libraries or
frameworks, follow these instructions to learn how to use OTel-ready libraries and instrumentation libraries for your dependencies.
```

Reply:
if I remember correctly we decided at some point that we say "app" not "application", maybe we need to add this to our "style guide", cc @chalin

Reply:
"App" is shorter, but it makes me think of a mobile app. It might be strange for a microservices developer?

Reply:
The general acceptance for "app" vs "application" can from when I was working on Google docs. I'm ok either way.

Reply:
let's keep "app" for now, this is again a separate discussion

Reply:
```suggestion
If you are instrumenting an app or service that use third-party libraries or
frameworks, follow these instructions to learn how to use natively instrumented libraries and instrumentation libraries for your dependencies.
```